apps/frontend/src/lib/backup/index.ts

```diff
@@ -473,3 +501,66 @@ export const createOAuthDeletionBackup = ({
     clientCreatedAt: new Date(),
   };
 };
+
+export interface CreateToggleSocialGraphBackupArgs {
+  email: string;
+  password: string;
+}
+
+export const createToggleSocialGraphBackup = ({
+    email,
+    password,
+  }: CreateToggleSocialGraphBackupArgs): CreateBackupData => {
+  const { authenticationTag, iv, encryptedData } = encryptBackupString({
+    backup: "",
+    email,
+    password,
+  });
+
+  return {
+    authenticationTag,
+    iv,
+    encryptedData,
+    backupEntryType: BackupEntryType.TOGGLE_SOCIAL_GRAPH,
+    clientCreatedAt: new Date(),
+  };
+};
+
+export interface CreateEdgeArgs {
```

Comment:
i think there should also be a notion here of "isTapSender"? seems like since edges are directed its important to know which side you are

Reply:
although i guess the server could provide you this information?

Reply:
actually this might be a better reason to store directionality in the backup. so technically this feature could be composable across different servers, i.e. you can upload your entire tap graph to a different server in the future

Reply:
added